examples/fungible/Cargo.toml

```diff
@@ -15,6 +15,10 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 thiserror = { workspace = true }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+linera-sdk = { workspace = true, features = ["test", "wasmer", "wasmtime"] }
```

Comment:
why both VMs?

Reply:
No reason I think. It should work with any one of them :+1: